thriftbp/prometheus.go

```diff
@@ -88,3 +88,93 @@ var (
 		Help: "The number of in-flight requests",
 	}, clientActiveRequestsLabels)
 )
+
+var (
+	serverConnectionsGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "thrift_server_connections",
```

Comment:
thriftbp?

Reply:
I try to keep consistent with other `thrift_server_` metrics, I think using `thriftbp_` here might cause confusions.

But if you think `thrift_server_` is only for things defined in baseplate spec and all other metrics should be namespaced by the package then that's fine by me as well.

Reply:
My thought is that `thriftbp` is for things that are specific to this thrift "framework" but might change if we make a different framework for thrift servers.  The `thrift_` ones must remain consistent across such changes, but that's a bit "easier" since there's a spec saying what they should be.

I don't really have a strong opinion though.

Reply:
that makes sense. changed to `thriftbp_`